src/screens/OrgList/OrgList.tsx

```diff
@@ -293,15 +344,31 @@ function orgList(): JSX.Element {
           </div>
           <div className={styles.btnsBlock}>
             <div className="d-flex">
-              <Dropdown aria-expanded="false" title="Sort organizations">
-                <Dropdown.Toggle variant="outline-success">
+              <Dropdown
+                aria-expanded="false"
+                title="Sort organizations"
+                data-testid="sort"
+              >
+                <Dropdown.Toggle
+                  variant="outline-success"
+                  data-testid="sortOrgs"
+                >
                   <SortIcon className={'me-1'} />
                   {t('sort')}
```

Comment:
The selected option text should appear here instead of just sort, and its variant should be of success